charts/kubefed/charts/controllermanager/templates/crds.yaml

```diff
@@ -579,9 +579,9 @@ spec:
               - resourceLock
               type: object
             scope:
-              description: The scope of the kubefed control plane should be either
-                `Namespaced` or `Cluster`. `Namespaced` indicates that the kubefed
-                namespace will be the only target for federation.
+              description: The scope of the KubeFed control plane should be either
+                `Namespaced` or `Cluster`.  `Namespaced` indicates that the KubeFed
```

Comment:
nit: i suspect @abrennan89 would have an opinion on two spaces vs. one here, but not a reason to hold the PR

Reply:
I'll fix in a follow-up.

Reply:
See #942